app/subscriber/src/features/home/home-filters/HomeFilters.tsx

```diff
@@ -18,6 +25,17 @@ export const HomeFilters: React.FC<IHomeFilterProps> = () => {
   const [{ filter }, { storeFilter }] = useContent();
   const [{ sources }] = useSources();
 
+  const [sourcesLookup, setSourcesLookup] = React.useState<ISourceModel[]>([]);
+
+  // if 304 is returned we need to get it from local storage
```

Comment:
I don't understand this.  304 means the content in the endpoint hasn't changed.   Which means you already have the latest content in cache.

You should have a lookup hook like we have for the editor app.  The difference is we don't want all the users being returned in the lookups.

Reply:
For some reason it was returning a blank array whenever it was 304, I will try out the lookup hook and maybe update the endpoints and see what happens